packages/core-contracts/src/interfaces/midas/IDepositVault.sol

```diff
@@ -0,0 +1,284 @@
+     * @return request id
+     */
+    function depositRequest(
+        address tokenIn,
+        uint256 amountToken,
+        bytes32 referrerId
+    ) external returns (uint256);
+
+    /**
+     * @notice Does the same as original `depositRequest` but allows specifying a custom tokensReceiver address.
+     * @param tokenIn address of tokenIn
+     * @param amountToken amount of `tokenIn` that will be taken from user (decimals 18)
+     * @param referrerId referrer id
+     * @param recipient address that receives the mTokens
+     * @return request id
+     */
+    function depositRequest(
+        address tokenIn,
+        uint256 amountToken,
+        bytes32 referrerId,
+        address recipient
+    ) external returns (uint256);
+
+    /**
+     * @notice approving requests from the `requestIds` array
+     * with the current mToken rate.
+     * Does same validation as `safeApproveRequest`.
+     * Mints mToken to request users.
+     * Sets request flags to Processed.
+     * @param requestIds request ids array
+     */
+    function safeBulkApproveRequest(uint256[] calldata requestIds) external;
+
+    /**
+     * @notice approving requests from the `requestIds` array using the `newOutRate`.
+     * Does same validation as `safeApproveRequest`.
+     * Mints mToken to request users.
+     * Sets request flags to Processed.
+     * @param requestIds request ids array
+     * @param newOutRate new mToken rate inputted by vault admin
+     */
+    function safeBulkApproveRequest(
+        uint256[] calldata requestIds,
+        uint256 newOutRate
+    ) external;
+
+    /**
+     * @notice approving request if inputted token rate fit price deviation percent
+     * Mints mToken to user.
+     * Sets request flag to Processed.
+     * @param requestId request id
+     * @param newOutRate mToken rate inputted by vault admin
+     */
+    function safeApproveRequest(uint256 requestId, uint256 newOutRate) external;
+
+    /**
+     * @notice approving request without price deviation check
+     * Mints mToken to user.
+     * Sets request flag to Processed.
+     * @param requestId request id
+     * @param newOutRate mToken rate inputted by vault admin
+     */
+    function approveRequest(uint256 requestId, uint256 newOutRate) external;
+
+    /**
+     * @notice rejecting request
+     * Sets request flag to Canceled.
+     * @param requestId request id
+     */
+    function rejectRequest(uint256 requestId) external;
+
+    /**
+     * @notice sets new minimal amount to deposit in EUR.
+     * can be called only from vault`s admin
+     * @param newValue new min. deposit value
+     */
+    function setMinMTokenAmountForFirstDeposit(uint256 newValue) external;
+
+    /**
+     * @notice sets new max supply cap value
+     * can be called only from vault`s admin
+     * @param newValue new max supply cap value
+     */
+    function setMaxSupplyCap(uint256 newValue) external;
+
+    function tokensConfig(
```

Comment:
Can we document these functions, even if it is to say that there is nothing to document? It stands out when reading the interface